src/environments/environment.ts

```diff
@@ -9,8 +9,8 @@ import env from './.env';
 export const environment = {
   production: false,
   version: env.npm_package_version + '-dev',
-  fineractPlatformTenantId: 'default',  // For connecting to server running elsewhere update the tenant identifier
-  baseApiUrl: 'https://demo.openmf.org',  // For connecting to server running elsewhere update the base API URL
+  fineractPlatformTenantId: 'staging',  // For connecting to server running elsewhere update the tenant identifier
```

Comment:
don't push environment.ts in your PR.

Reply:
> don't push environment.ts in your PR.

Okay, I will make the necessary adjustments.

Reply:
@DivineWisdom I think you are using git add . to add your changes. I suggest you to use git add address of the source folder in which you have made changes and then the unnecessary changes will not be pushed. example here you can use git add src/app/organization as you are making changes to the organization folder.